share/translations/measurements_nl_NL.ts

```diff
@@ -3451,19 +3451,19 @@
         <location filename="../../src/libs/vpatterndb/vtranslatemeasurements.cpp" line="1713"/>
         <source>armscye_width</source>
         <comment>Name in a formula. Don&apos;t use math symbols and space in name!!!!</comment>
-        <translation>Arminzet_wijdte</translation>
+        <translation>Armsgat_breedte</translation>
```

Comment:
all other formula names start with lowercase, maybe adapt this as well (while at it)

Reply:
You are absolutely right that these must also be corrected. I must have copied the capitalization from the old translation without thinking critically enough. It is quite difficult to remain fully consistent when translating many terms!
I will correct the file and resubmit, but probably not tonight any more.